src/hooks/useConsoleErrors.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-param-reassign */
+const useConsoleErrors = (sections, activeSections, zeroStateSections) => {
```

Comment:
since nothing in here requires hooking into anything special about react component life-cycles, a vanilla javascript function makes more senes for us here. 

more detail on this here: https://www.memberstack.com/blog/react-custom-hooks#:~:text=Custom%20hooks%20are%20helpful%20utilities,that%20can%20be%20used%20anywhere.

With this in mind, I recommend that we move this function to `src/utils.ts` and give it a name that doesn't start with 'use'. Something like `validateSectionsDataFormat`, `validateSectionsDataModels`, etc.